content/en/blog/2022/opamp/index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Using OpenTelemetry OpAMP to modify service telemetry on the go
+linkTitle: OpAMP service usage
+date: 2022-10-12
+author: >-
+  [Nimrod Shlagman](https://github.com/nemoshlag) (Cisco)
+---
+
+How verbose should your service telemetry be? Should a service output all
+traces, metrics, and logs 100% of the time? How much of the service traffic
+should be sampled? I would like to suggest the answer of “it depends”. Desired
+telemetry data differs in a service lifecycle from development to continuous
+deployment. It changes when a client faces an error or our service has been
+thrown into major scale. In an ideal world, we would modify the entire service’s
```

Comment:
>  In an ideal world

I have my issues with that statement, ideal compared to what? I think it would be helpful to go into a little bit more detail about "we would modify the entire service’s telemetry without the need for code changes or deployment."

So maybe start a new paragraph and outline the problem that right now with OpenTelemetry you can not do that, and what the implications are (=> service restart, no capability to change behavior on runtime, etc.)